release-notes/template/valtimo-backend-libraries.md

```diff
@@ -4,10 +4,10 @@
 
```

Comment:
This documentation should go to the 9.26.0 release notes. I'm not if we've a branch for that yet.

Reply:
I don't see a branch 9.26.0. Do you want me to make one? If yes, what will be the base?

Reply:
The branch was just created

Reply:
Okay thanx Klaas. I will make a new pr.